usb.js

```diff
@@ -1,8 +1,6 @@
-var binary = require('node-pre-gyp');
-var path = require('path');
-var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var bindings = require('bindings')('usb_bindings.node');
 
-var usb = exports = module.exports = require(binding_path);
+var usb = exports = module.exports = bindings;
```

Comment:
Since that `bindings` is never used after this, you can consolidate: 

```js
var usb = exports = module.exports = require('bindings')('usb_bindings.node');
```